addons/knobs/README.md

```diff
@@ -63,6 +64,28 @@ stories.add('as dynamic variables', () => {
 });
 ```
 
+### With Angular
+```js
+import { storiesOf } from '@storybook/angular';
+import { boolean, number, text, withKnobs } from '@storybook/addon-knobs/angular';
+
+import { Button } from '@storybook/angular/demo';
+
+const stories = storiesOf('Storybook Knobs', module);
+
+// The "addDecorator" must come before the "add"
```

Comment:
I would say 

"withKnobs" decorator should be applied **before** the stories using knobs